File: backend/RawDataWorkSheet/RawDataWorkSheet/Models/Requests/WorksheetLogRequest.cs
namespace RawDataWorkSheet.Models.Requests
{
    public class WorksheetLogRequest
    {
        public string? WorksheetId { get; set; }
        public int? ParameterId { get; set; }
        public string? Remarks { get; set; }
        public string? Action { get; set; }
        public string? EmployeeId { get; set; }
        public string? Role { get; set; }
        public string? ReferenceType { get; set; }
        public string? ReferenceId { get; set; }
    }
}

File: frontend/src/models/WorksheetLogRequest.ts

export interface WorksheetLogRequest {
    worksheetId?: string | null;
    parameterId?: number | null;
    remarks?: string | null;
    action: string;
    employeeId: string;
    role: string;
    referenceType?: string | null;
    referenceId?: string | null;
}
